### PR TITLE
ENDOC-447 Change host names to single

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Here are a few common questions about the quickstart environment. The quickstart
 ### General
 **Q: How can I remove a quickstart environment?** 
 
-**A:** If you want to completely remove the VM created by Multipass, you can use `multipass delete <VM-NAME>` (where the default VM-NAME for a quickstart is `entando`) and then `multipass purge` to recover the resources. If you just want to shutdown Entando but keep the VM, you can use `multipass shell <VM-NAME>` to shell into the VM and then remove the namespace via `sudo kubectl delete namespace entando`. 
+**A:** If you want to completely remove the VM created by Multipass, you can use `multipass delete YOUR-VM-NAME` (where the default VM-NAME for a quickstart is `entando`) and then `multipass purge` to recover the resources. If you just want to shutdown Entando but keep the VM, you can use `multipass shell YOUR-VM-NAME` to shell into the VM and then remove the namespace via `sudo kubectl delete namespace entando`.
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/local-tips-and-tricks.md:14]: missed one VM-NAME in the middle

File: vuepress/docs/next/tutorials/create/mfe/communication.md [hunk 2]
```diff
@@ -275,7 +275,7 @@ You’ve now created a micro frontend that listens to custom events.
 
 Now let's add the publisher and subscriber micro frontends in Entando.
 
-> Note: These are the same steps as the `Create a React Micro Frontend` tutorial.
+> Note: These follow the same steps as in the [Create a React Micro Frontend](./react.html#build-it) tutorial.
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/create/mfe/communication.md:278]: .html -> .md

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md [hunk 3]
```diff
@@ -63,23 +63,21 @@ The namespace will be recreated, preserving the images already pulled, so it's u
...
+quickstart.192.168.99.1.nip.io
+YOUR-APP.192.168.99.1.nip.io
 ```
-The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
+The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando-app.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments, there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/local-tips-and-tricks.md:71]: We don't have the ENTANDO_DEFAULT_ROUTING_SUFFIX anymore, maybe just change to this: 
...`ingressHostname` (e.g. in your entando.yaml) is based on an IP address that is created during the initial VM installation.

**Review by nshaw — CHANGES_REQUESTED** [review]

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md [hunk 4]
```diff
@@ -201,7 +197,7 @@ python3 -m http.server 8000
 13. You should now be able to install Entando using the static IP. If your Entando installation stalled during startup, and was previously configured with a static IP, it should continue initializing as soon as the external address is functional again. 
 
 #### Option 4: Reinstall Entando
-We're including this option because it works and requires no additional configuration. If you plan to regularly work with Entando we recommend developing against a centralized and shared Kubernetes instance rather than running a full stack locally. If you require a local cluster we recommend using option 1 or 2.
+We're including this option because it works and requires no additional configuration. If you work with Entando regularly, we recommend developing in a centralized and shared Kubernetes instance rather than running a full stack locally. If you require a local cluster, we recommend using option 1 or 2.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/local-tips-and-tricks.md:200]: @nshaw can you comment on the use of "against" vs "in" here?

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md [hunk 5]
```diff
@@ -63,23 +63,21 @@ The namespace will be recreated, preserving the images already pulled, so it's u
...
 ```
- 192.168.99.1 quickstart-kc-entando.192.168.99.1.nip.io
- 192.168.99.1 quickstart-eci-entando.192.168.99.1.nip.io
- 192.168.99.1 quickstart-entando.192.168.99.1.nip.io
+ 192.168.99.1 quickstart.192.168.99.1.nip.io
+ 192.168.99.1 YOUR-APP.192.168.99.1.nip.io
 ```
 - If you add microservices to your installation, you may need to add additional mappings for the new ingresses.
-- See [this section below](#option-2-manually-update-your-hosts-file) for detailed steps to perform this on Windows.
+- To update files on a Windows machine, see the [Windows process](#option-2-manually-update-your-hosts-file) below.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/local-tips-and-tricks.md:80]: on Windows (nix machine - that gets into tricky territory with emulators, etc. safer to confine references to the OS layer.)

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md [hunk 6]
```diff
@@ -63,23 +63,21 @@ The namespace will be recreated, preserving the images already pulled, so it's u
...
+The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando-app.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments, there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
 
 ### `.nip.io isn't allowed`
- - This could be due to firewall settings or corporate security policies. The simplest workaround is to manually edit your /etc/hosts file and map the necessary domains to the IP of your local virtual machine.
+ - This could be due to firewall settings or corporate security policies. The simplest workaround is to manually edit your /etc/hosts file and map the necessary domain to the IP of your local virtual machine.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/local-tips-and-tricks.md:74]: just one domain? if just one can omit "necessary"

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md (hunk 5 above)
Comment (nshaw) [reply vuepress/docs/next/docs/reference/local-tips-and-tricks.md:80]: Agreed, just say "on Windows"

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md (hunk 4 above)
Comment (nshaw) [reply vuepress/docs/next/docs/reference/local-tips-and-tricks.md:200]: @Lyd1aCla1r3 I think "in" is better.

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md (hunk 6 above)
Comment (nshaw) [reply vuepress/docs/next/docs/reference/local-tips-and-tricks.md:74]: Agreed. Just one domain now.

File: vuepress/docs/next/docs/reference/local-tips-and-tricks.md (hunk 4 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/reference/local-tips-and-tricks.md:200]: @nshaw i was wondering what it meant to develop against an environment... is that common phrasing?

Comment (nshaw) [reply vuepress/docs/next/docs/reference/local-tips-and-tricks.md:200]: @Lyd1aCla1r3 it's not that unusual. The only comparison I can think of is when you develop "against" or "towards" a standard, basically with a goal of meeting a target. I do think "in" is a bit better but just personal opinion/preference on this one. We log in to or  shell/ssh into a VM, we log on to a cloud provider, we deploy into Entando, pods run in Kubernetes...

**Review by nshaw — APPROVED** [review]

**Pull request merged**